### PR TITLE
Srcs 170 wind sensor node

### PR DESCRIPTION
**Pull request by JSpringett**

### CODE REVIEW

File: NodeIDs.h [hunk 1]
```diff
@@ -16,6 +16,7 @@
 enum class NodeID {
```
Comment (simchanu29) [inline NodeIDs.h:20]: A quick description of NodeID would be best.

File: Messages/MessageTypes.h [hunk 2]
```diff
@@ -15,5 +15,6 @@
 
 
 enum class MessageType {
```
Comment (simchanu29) [inline Messages/MessageTypes.h:19]: Could you add a quick description of the purpose of this class and how to fill it in the future ? I understood this is where you store the message types, but it's not obvious.

I didn't understand either why is there DataRequest = 0 inside the message types.

File: NodeIDs.h (hunk 1 above)
Comment (JSpringett) [inline NodeIDs.h:20]: Done

File: Nodes/CV7Node.cpp [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+CV7Node::CV7Node(MessageBus& msgBus, std::string portName, unsigned int baudRate)
+	:ActiveNode(NodeID::WindSensor, msgBus), m_Initialised(false), m_fd(-1), m_PortName(portName), m_BaudRate(baudRate),
+	 m_MeanWindDir(DATA_OUT_OF_RANGE), m_MeanWindSpeed(DATA_OUT_OF_RANGE), m_MeanWindTemp(DATA_OUT_OF_RANGE)
+{
+
+}
+
+bool CV7Node::init()
+{
```
Comment (simchanu29) [inline Nodes/CV7Node.cpp:40]: What happens if the success is false while the m_Initialised is true ? The CV7Node::WindSensor start with a failed initialization ?

File: Messages/MessageTypes.h (hunk 2 above)
Comment (JSpringett) [inline Messages/MessageTypes.h:19]: DataRequest = 0 Means that when you convert it to a integer that the enum values will start from 0 and count up. I tend to do it by habit. I've also added a better description to the file description.

File: Nodes/CV7Node.cpp (hunk 3 above)
Comment (JSpringett) [inline Nodes/CV7Node.cpp:40]: Fixed

**Comment by sailingrobots** [comment]
Look's alright to me

//Emil

**Comment by pophaax** [comment]
approved

**Comment by pophaax** [comment]
approved (Elouan)

**Comment by OliverNyholm** [comment]
Can't see anything faulty. I approve

**Pull request merged**